anyric/super-record
# #169227868 Update Select fields to make them Editable

#### What does this PR do?
This PR update the select fields to make them editable 

#### Description of Task to be completed?
- Update the select fields to make them editable

#### How should this be manually tested?
- Clone this branch
- In the root directory run `python manage.py runserver`
- Navigate to `http://127.0.0.1:8000/` to login
- Click on the `Cart` or `Expense` menu from the dashboard
- Type an item name into the item or category field to see the filter

#### Any background context you want to provide?
Currently the select lists are not editable and one can only scroll to select the required items. This can become extremely long with more data populated in the database. This should therefore be made editable for users to be able to filter the items as they type required values in the field.

#### What are the relevant pivotal tracker stories?
[#169227868](https://www.pivotaltracker.com/story/show/169227868)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29800415/67104563-d4aad300-f1cf-11e9-854e-eb95974bcf2c.png)


#### Questions:
N/A
